### DISCUSSION TITLE
Connecting ComfyUI with n8n

### DISCUSSION THREAD

**Discussion by AnirudhG07** [Comfy-Org/ComfyUI#6133] [Q&A]
Hi. Does anyone know how to connect ComfyUI with n8n? I coudnt find resources for this on net. I would like to generate images through comfyUI but automate it with n8n. Anyone has any ideas, tips on how to do it and if its possible or not.

**Comment by vutran8902** [comment]
interested as well

**Comment by looknear** [comment]
there is a node to activate a workflow of comfyui... but....
all previous inputs - are not accessible after this node...
and no simple way to pass some global value to all nodes... like a path to save images...
n8n just not robust enough. not stable enough. too many restarts needed, too many unstable bugs.

**Comment by vishaldwdi** [comment]
I am currently expermenting with it. https://www.perplexity.ai/search/can-i-connect-comfyui-to-n8n-h-3wFwRrBNQlOvuaFwGpQwgw#0
